wordpress-mobile/WordPress-Android
# RI: Show a list of "chips" with interests on the UI

Master issue: #12027

This PR just gets mocked list of interests from `ReaderTagRepository` and displays them as list of "chips" on the UI. 

### To test:
0. Enable runtime feature flag as described in #12050 and switch to build variant where it is enabled.
1. Go to Reader tab.
2. Wait for a while.
3. Notice that list of "chips" with interests is shown on the UI.

![device-2020-06-02-135659](https://user-images.githubusercontent.com/1405144/83498027-fba4e380-a4d8-11ea-95a8-b83de41aa1e1.png)

Note:
Still need to refine the "chips" UI (font, colors etc), will do in upcoming PRs.

PR submission checklist:
